src/MailerSendTrait.php

```diff
@@ -8,16 +8,25 @@
 
 trait MailerSendTrait
 {
-    public function mailersend(string $template_id = null, array $variables = [], array $tags = [], array $personalization = [])
+    public function mailersend(
+        string $template_id = null,
+        array $variables = [],
+        array $tags = [],
+        array $personalization = [],
+        ?bool $precedenceBulkHeader = null,
+        ?int $sendAt = null
```

Comment:
Let's make it more user friendly and allow users to pass a time instance, which we convert to a unix timestamp when sending the request to API